### ISSUE TITLE
[Question]: 

### ISSUE THREAD

**Issue by elearningdienst** [danielmiessler/fabric#319]
### What is your question?

After reading the documentation, I am still not sure how to get the output in a language other than English. I tried changing the patterns manually by adding `"- Output always in German language, no matter what input language is given"`, but that is not very convenient. 

Is there a way to do this in the `~/.config/fabric/.env` file or during setup? Maybe by adding the preferred output language, or translating the output to another language after generating

**Comment by snafu4** [comment]
Create a "context.md" file with "always output in German" (no quotes).  Put it in "~/.config/fabric".  Worked for me.

**Comment by elearningdienst** [comment]
Maybe - if that is not implemented yet it could be an enhancement. `fabric -l` or `fabric --language`  with the ISO Language Code as option. e.g. f`abric -l de` or an option like `--changeDefaultOutputLanguage de`

**Comment by xssdoctor** [comment]
Right now, the only way to do this is what @snafu4 mentioned. Make a context.md file and write that you want everything outputted in german

**Comment by christrille** [comment]
Is it possible to create various "context.md" files for different languages, and call them with different options in the 'fabric 'line.
Something like: 
- context_us.md. => -c_us
- context_spain.md. => -c_es

**Comment by Gerkinfeltser** [comment]
Yeah, I was curious about multiple contexts as well. Not multiple languages in my case but I'm sure this would be a common usecase as well. I was curious if there was the possibility of having a setting that would automatically include a default context.md too (but this might warrant a separate issue).
